spec/user_interface_spec.rb

```diff
@@ -22,6 +23,34 @@
     end
   end
 
+  describe 'who_goes_first?' do
+    it 'asks player who should go first' do
+      allow(user_interface).to receive(:gets).and_return('1')
+      expect(STDOUT).to receive(:puts).with(first_player)
+      user_interface.who_goes_first?
+    end
+
+    it 'returns the user selection' do
+      allow(STDOUT).to receive(:puts)
+      allow(user_interface).to receive(:gets).and_return('1')
+      allow(STDOUT).to receive(:puts)
+      expect(user_interface.who_goes_first?).to eql('1')
+      allow(STDOUT).to receive(:puts).twice
+      allow(user_interface).to receive(:gets).and_return('2')
+      expect(user_interface.who_goes_first?).to eql('2')
```

Comment:
Can you refactor the user interface so that we do not have to use the mocking API?